src/matrixrtc/CallMembership.ts

```diff
@@ -91,7 +91,7 @@ export class CallMembership {
     }
 
     public isExpired(): boolean {
-        return this.getAbsoluteExpiry() < this.parentEvent.getTs() + this.parentEvent.getLocalAge();
+        return this.getMsUntilExpiry() <= 0;
```

Comment:
this seems unrelated to the subject of this PR "Better `localTimestamp` fallback"? It looks like its switching the expiry calculation from being based on `origin_server_ts` to being based on `age`?

Which might be a reasonable thing to do, but it's not what the PR says it does?

Reply:
It really is the exact same thing just a nicer way to write it. (so it is just a cosmetic change)
(except the `<` -> `<=` which is not really chanign anything considering the epxiration is more or less arbitrary and could very well be 35999 instead of 36000)

Let me take the time to show the equivalence:
```
getMsUntilExpiry() <= 0

using:
    public getMsUntilExpiry(): number {
        return this.getLocalExpiry() - Date.now();
    }

getLocalExpiry() - Date.now() <= 0

using:
public getLocalExpiry(): number {
    const relativeCreationTime = this.parentEvent.getTs() - this.createdTs();
    const localCreationTs = this.parentEvent.localTimestamp - relativeCreationTime;
    return localCreationTs + this.data.expires;
}

localCreationTs + data.expires - Date.now() <= 0
parentEvent.localTimestamp - relativeCreationTime + data.expires - Date.now() <= 0
parentEvent.localTimestamp - (parentEvent.getTs() - createdTs()) + data.expires - Date.now() <= 0
parentEvent.localTimestamp - parentEvent.getTs() + createdTs() + data.expires - Date.now() <= 0
createdTs() + data.expires <= + parentEvent.getTs() + Date.now() - parentEvent.localTimestamp

using:
    public getAbsoluteExpiry(): number {
        return this.createdTs() + this.data.expires;
    }

getAbsoluteExpiry()  <=  + parentEvent.getTs() + (Date.now() - parentEvent.localTimestamp)

using:
    public getLocalAge(): number {
        return Date.now() - this.localTimestamp;
    }

getAbsoluteExpiry() <= parentEvent.getTs() + parentEvent.getLocalAge();
```

Reply:
Ah yes that seems right! Thanks for taking the time to explain so clearly.